main.cpp

```diff
@@ -49,77 +49,105 @@ Create a branch named Part1
  
  Send me the the link to your repl.it in a DM on Slack
 
- Wait for my code review.
+ Wait for my code review...
  */
 
 #include <iostream>
 #include <string>
 struct T
 {
-    T(<#type name#> v, const char* <#variable name#>)   //1
-    //2
-    //3
+    float value;
+    const std::string name; 
+    
+    T (float v, const char* aCharacter) :
+        value (v),
+        name (aCharacter)
+    { }
 };
 
-struct <#structName1#>                                //4
+struct Operation 
 {
-    <#type name#> compare(<#type name#> a, <#type name#> b) //5
-    {
-        if( a->value < b->value ) return a;
-        if( a->value > b->value ) return b;
-        return nullptr;
-    }
+    T* compare(T* a, T* b)
+    {   
+        if (a != nullptr && b != nullptr) 
+        {
+            if( a->value < b->value ) return a;
+            if( a->value > b->value ) return b;
+        }
+        return nullptr; 
+    } 
 };
 
 struct U
 {
-    float <#name1#> { 0 }, <#name2#> { 0 };
-    <#returnType#> <#memberFunction#>(<#type name#>* <#updatedValue#>)      //12
+    float var1 { 0 }, var2 { 0 };
+    
+    U(float a) : var1 (a) { }
+    
+    U();
```

Comment:
Remove these member functions.

Reply:
without the first constructor, it won't compile...